README.md

```diff
@@ -109,4 +122,4 @@ Control stick roll left (+y rotation) -> model visually moves in -z rotation
 
 Onshape space:
 +x rotation -> -z
-+z rotation -> -x result
\ No newline at end of file
++z rotation -> -x result
```

Comment:
Very small nitpick - in the future, try to avoid committing formatting changes elsewhere in the file to keep the history clean and minimize merge conflicts. In GitHub Desktop you can uncheck certain lines of code to not commit them, and there are probably ways do this with other clients.